### ISSUE TITLE
theme_foundation not working with ggplot2 v2.0.0

### ISSUE THREAD

**Issue by KJByron** [jrnold/ggthemes#56]
my old custom themes stopped working when I upgraded to ggplot2 v2.0.0, suspect the trouble is within theme_foundation

here is a simple example:
theme_cust <- function (base_size = 12, base_family = "sans") 
{
	ggthemes::theme_foundation(base_size = base_size, base_family = base_family) + 
			ggplot2::theme(
				panel.border = element_rect(fill = NA, color = "red", linetype = "solid", size = 2))
}

theme_cust2 <- function (base_size = 12, base_family = "sans") 
{
	ggplot2::theme_grey(base_size = base_size, base_family = base_family)  %+replace%  
			ggplot2::theme(
				panel.border = element_rect(fill = NA, color = "red", linetype = "solid", size = 2))
}

p <- ggplot(iris, aes(Species, Sepal.Length)) +
	geom_boxplot() 
p + theme_cust()       # error
p + theme_cust2()     # fine

thanks

**Comment by jrnold** [comment]
The issue seems to be that I changed how I created `theme_foundation`. Now it removes all `colour` and `fill` values from `theme_gray`. This means the the value of `fill` and `colour` for `element_rect` were `NULL`. I fixed this so the top level elements (`rect`, `line`, `text`) have `colour = "black"` and `fill = "white"`) as before. It'll be a little bit before this is up on CRAN, but it is fixed in the github version.
```r
devtools::install_github("jrnold/ggthemes")
```
